<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiClient</class>
 <widget class="QMainWindow" name="MultiClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multicast Client</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="welcomeLabel">
               <property name="text">
                <string>Welcome to the Multicast Client for CommAudio!
Enter in the IP and Port of the server,
and click connect to begin listening.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="ConnectionControls">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>250</height>
                </size>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="title">
                <string>Connection Controls</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <property name="leftMargin">
                 <number>7</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>7</number>
                </property>
                <item>
                 <widget class="QGroupBox" name="IPControls">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>IP Address</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_8">
                   <property name="topMargin">
                    <number>5</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="lineIPAddress">
                     <property name="text">
                      <string>234.7.8.9</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QGroupBox" name="PortControls">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>60</height>
                   </size>
                  </property>
                  <property name="title">
                   <string>Port</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <property name="topMargin">
                    <number>5</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="linePort">
                     <property name="text">
                      <string>7000</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <property name="leftMargin">
                   <number>11</number>
                  </property>
                  <property name="rightMargin">
                   <number>11</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="buttonConnect">
                    <property name="text">
                     <string>Connect</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="buttonDisconnect">
                    <property name="text">
                     <string>Disconnect</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12" stretch="0,0">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <widget class="QGroupBox" name="AudioControls">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="title">
               <string>Audio Controls</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <item row="0" column="0">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <item>
                    <widget class="QPushButton" name="buttonPlay">
                     <property name="text">
                      <string>Play</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="buttonStopAudio">
                     <property name="text">
                      <string>Stop</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_5">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>50</height>
                  </size>
                 </property>
                 <property name="title">
                  <string>Sound</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QSlider" name="sliderSound">
                    <property name="maximum">
                     <number>100</number>
                    </property>
                    <property name="pageStep">
                     <number>1</number>
                    </property>
                    <property name="value">
                     <number>100</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="StatusBar">
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>364</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
